src/loaders/common/loader_common_names.py

```diff
@@ -45,6 +45,8 @@
 SYS_TRAIT_ID = 'trait_id'  # unique identifier for a trait
 DETECTED_GENE_SCORE_COL = 'detected_genes_score'  # column name for the detected genes score
 
+CELL_ID = 'cell_id'  # data id field for CellDetail object
```

Comment:
This should go in the heatmap models module along with the other field names

Reply:
👍

Reply:
:+1: